app/views/layouts/application.html.erb

```diff
@@ -1,35 +1,37 @@
 <!DOCTYPE html>
 <html>
-<head>
```

Comment:
You seem to have lost your opening `<head>` tag. That's going to cause lots of issues